app/build.gradle

```diff
@@ -23,16 +24,18 @@ android {
     }
 }
 
+ext.support_lib_version = '25.3.1'
+
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
-    compile 'com.android.support:appcompat-v7:25.3.1'
-    compile 'com.android.support:support-v13:25.3.1'
-    compile 'com.android.support:design:25.3.1'
-    compile 'com.android.support:support-v4:25.3.1'
-    compile 'com.android.support:cardview-v7:25.3.1'
+    compile "com.android.support:appcompat-v7:$support_lib_version"
```

Comment:
please don't do this. prefer using explicit names separately. studio updates them automatically that way.